rns_shell/platform/graphics/x11/WindowX11.cpp

```diff
@@ -58,13 +63,38 @@ void Window::createEventLoop(Application* app) {
             while (count-- && !done) {
                 XEvent event;
                 XNextEvent(display, &event);
-
                 switch (event.type) {
-                    case ConfigureNotify:
-                        RNS_LOG_INFO("Resize Request with (Width x Height) : (" << event.xconfigurerequest.width <<
+                    case ConfigureNotify:{
+
+                        if(event.xconfigurerequest.window == DefaultRootWindow(display)) {
+                            RNS_LOG_INFO(" ROOT Window(Screen) Size :" << event.xconfigurerequest.width << "x" << event.xconfigurerequest.height);
+
+                            if(((RnsShell::PlatformDisplay::sharedDisplay().getCurrentScreenSize().width()!=event.xconfigurerequest.width) ||
```

Comment:
To avoid calling same function multiple times here, can it be called once and saved in local variable and used